srcpkgs/encrypted-dns/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'encrypted-dns'
+pkgname=encrypted-dns
+version=0.9.13
+revision=1
+build_style=cargo
+makedepends="libsodium-devel"
+short_desc="DNSCrypt v2 server with support for DNSSEC and DoH forwarding"
+maintainer="MeganerdNL <meganerd@meganerd.nl>"
+license="MIT"
+homepage="https://github.com/DNSCrypt/encrypted-dns-server"
+distfiles="https://github.com/DNSCrypt/encrypted-dns-server/archive/refs/tags/${version}.tar.gz"
+checksum=5e4f9143313bf58888c31ec4e220e9fb65b28b60fe5b6aff872f9f2ecb7537d5
+
+system_accounts="_encrypted_dns"
+_encrypted_dns_homedir="/var/lib/encrypted-dns"
+
+conf_files="/etc/encrypted-dns/encrypted-dns.toml"
+
+mutable_files="/var/lib/encrypted-dns/undelegated.txt"
+
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	XBPS_CROSS_RUSTFLAGS+=" -latomic"
+	makedepends+=" libatomic-devel"
+fi
+
+case "$XBPS_TARGET_MACHINE" in
+	*-musl) makedepends+=" musl-devel libstdc++";;
```

Comment:
why is this needed?

Reply:
> why is this needed?

It won't compile on musl otherwise. 
Complains of missing dependencies that are not there on glibc.

Reply:
works fine with a musl masterdir:

```
$ ./xbps-src -m masterdir-musl -a armv6l-musl pkg encrypted-dns
...
=> Registering new packages to /host/binpkgs/encrypted-dns
index: added `encrypted-dns-0.9.13_1' (armv6l-musl).
index: 1 packages registered.
```

Reply:
Not reproducible.  Also failing on Github. See comment in main section.